rero_ils/modules/item_types/jsonschemas/item_types/item_type-v0.0.1.json

```diff
@@ -90,6 +94,124 @@
         }
       }
     },
+    "negative_availability": {
+      "type": "boolean",
+      "title": "Mark the items of this category as not available",
+      "description": "If enabled, all items of this circulation category will be displayed as not available, with the red availability symbol.",
+      "default": false
+    },
+    "displayed_status": {
+      "type": "array",
+      "title": "Displayed status",
+      "description": "Short text displayed in the interface, for instance \"in acquisition\" or \"missing\".",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "title": "Status",
+        "additionalProperties": false,
+        "required": [
+          "language",
+          "label"
+        ],
+        "propertiesOrder": [
+          "language",
+          "label"
+        ],
+        "properties": {
+          "language": {
+            "$ref": "https://ils.rero.ch/schemas/common/ui-languages-v0.0.1.json#/language"
+          },
+          "label": {
+            "type": "string",
```

Comment:
description?

Reply:
Can you check the order of the properties in the JSONSchema: https://github.com/rero/developer-resources/blob/master/data/jsonschema.md